### PR TITLE
Fix resource query

### PR DESCRIPTION
**Pull request by Fluder-Paradyne**
### Description

temp change to query resource so we can use query resource without creating summary

### Solution and Design

The summary is replaced with file names

### Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update

### Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have read the contributing guide and my code conforms to the guidelines.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have added the required tests.

### CODE REVIEW

File: superagi/resource_manager/resource_summary.py [hunk 1]
```diff
@@ -58,29 +58,30 @@ def generate_agent_summary(self, agent_id: int, generate_all: bool = False) -> s
 
         summary_texts = [resource.summary for resource in resources if resource.summary is not None]
 
+        resource_summary = " ".join([resource.name for resource in resources])
```
Comment (TransformerOptimus) [inline superagi/resource_manager/resource_summary.py:61]: update the base prompt.

**Review by TransformerOptimus — APPROVED** [review]

File: superagi/resource_manager/resource_summary.py [hunk 2]
```diff
@@ -33,14 +33,14 @@ def add_to_vector_store_and_create_summary(self, agent_id: int, resource_id: int
...
             logger.error("add_to_vector_store_and_create_summary: Unable to save document to vector store.", e)
-        summary = None
-        try:
-            summary = LlamaDocumentSummary(model_api_key=model_api_key).generate_summary_of_document(documents)
-        except Exception as e:
-            logger.error("add_to_vector_store_and_create_summary - Unable to generate summary of document.", e)
-        resource = self.session.query(Resource).filter(Resource.id == resource_id).first()
-        resource.summary = summary
-        self.session.commit()
+        # summary = None
```
Comment (TransformerOptimus) [inline superagi/resource_manager/resource_summary.py:36]: remove these commented lines.

File: superagi/resource_manager/resource_summary.py [hunk 3]
```diff
@@ -58,29 +58,30 @@ def generate_agent_summary(self, agent_id: int, generate_all: bool = False) -> s
...
-            file_paths = [resource.path for resource in resources if resource.summary is None]
-            for file_path in file_paths:
-                if resources[0].storage_type == 'S3':
-                    documents = ResourceManager(str(agent_id)).create_llama_document_s3(file_path)
-                else:
-                    documents = ResourceManager(str(agent_id)).create_llama_document(file_path)
-                if documents is not None and len(documents) > 0:
-                    summary_texts.append(LlamaDocumentSummary(model_api_key=model_api_key, model_source=model_source).generate_summary_of_document(documents))
-
+        # if len(summary_texts) < len(resources) and generate_all:
```
Comment (TransformerOptimus) [inline superagi/resource_manager/resource_summary.py:64]: remove these commented lines.

File: superagi/resource_manager/resource_summary.py [hunk 4]
```diff
@@ -58,29 +58,30 @@ def generate_agent_summary(self, agent_id: int, generate_all: bool = False) -> s
...
                    AgentConfiguration.key == "last_resource_time").first()
-        if agent_last_resource is not None and \
-                datetime.strptime(agent_last_resource.value, '%Y-%m-%d %H:%M:%S.%f') == resources[-1].updated_at \
-                and not generate_all:
-            return
-
-        resource_summary = summary_texts[0] if summary_texts else None
-        if len(summary_texts) > 1:
-            resource_summary = LlamaDocumentSummary(model_api_key=model_api_key).generate_summary_of_texts(summary_texts)
+        # if agent_last_resource is not None and \
```
Comment (TransformerOptimus) [inline superagi/resource_manager/resource_summary.py:77]: remove these commented lines.

**Review by boundless-asura — APPROVED** [review]

**Pull request merged**